### DISCUSSION TITLE
posts/2022-03-09-tidyoutliers020/

### DISCUSSION THREAD

**Discussion by giscus** [brunocarlin/carlin#11] [General]
# posts/2022-03-09-tidyoutliers020/

The tidy.outliers package now has 5 ways  of scoring outliers on your data!

https://carlin-blog.netlify.app/posts/2022-03-09-tidyoutliers020/

**Comment by opus1993** [comment]
Nice work.  I'm interested in giving the package a try over on Kaggle in the tabular playground to explore what it is capable of.

**Reply by brunocarlin** [reply]
Oh nice! Good luck! Let me know if something bugs out

**Reply by opus1993** [reply]
several challenges.  I'll fork it, build it locally, and look a little closer
